### PR TITLE
Update INI format description

### PR DESCRIPTION
**Pull request by Argent77**

### CODE REVIEW

**Comment by Argent77** [comment]
The PR should be ready for merging now.

File: file_formats/ie_formats/ini.htm [hunk 1]
```diff
@@ -332,60 +335,61 @@
...
+        <strong>hold_selected_point_key = boolean_value</strong><br />
+        This attribute prevents the engine from updating the currently selected spawn location index directly after the spawn operation (post-selection). If "create_qty" is greater than 1 then this attribute forces the engine to reuse the initially selected spawn location for all spawns created during the current spawn operation. This attribute affects only sequential "point_select" modes which includes <span class="code">i</span>, <span class="code">r</span> and <span class="code">s</span>.<br />
         <br />
 
-        <strong>check_by_view_port = ???</strong><br />
-        The meaning of this attribute is not fully understood, but when on it limits spawning to outside of the current viewport (part of the area currently displayed). <i>ignore_can_see</i> also has to be set and at least in PSTEE, the spawnpoint has to be within visual range. It's unused in the originals under this name, but there are plenty of uses of <i>check_view_port</i> </span><br />
+        <strong>check_by_view_port = boolean_value</strong><br />
+        This attribute appears to be unimplemented.<br />
```
Comment (lynxlynxlynx) [inline file_formats/ie_formats/ini.htm:392]: This should still mention check_view_port, since there are so many uses.

Comment (Argent77) [reply file_formats/ie_formats/ini.htm]: Done.

File: file_formats/ie_formats/ini.htm [hunk 2]
```diff
@@ -389,7 +389,7 @@
         <br />
 
         <strong>check_by_view_port = boolean_value</strong><br />
-        This attribute appears to be unimplemented.<br />
+        This attribute appears to be unimplemented. <strong>Note:</strong> There are plenty of uses of the <em>check_view_port</em> attribute which is most likely unimplemented as well.<br />
```
Comment (lynxlynxlynx) [inline file_formats/ie_formats/ini.htm:392]: So this something that Bubb didn't cover? Then please ask him for this final detail.

Comment (Argent77) [reply file_formats/ie_formats/ini.htm]: Confirmed that _check_view_port_ is not supported by the engine at all.

**Comment by lynxlynxlynx** [comment]
Thank you!

**Pull request merged**